service/src/main/java/bio/terra/workspace/db/WorkspaceDao.java

```diff
@@ -128,9 +138,7 @@ public Workspace getWorkspace(UUID id) {
     if (id == null) {
       throw new MissingRequiredFieldException("Valid workspace id is required");
     }
-    String sql =
-        WORKSPACE_SELECT_SQL
-            + " WHERE W.workspace_id = :id AND (C.cloud_platform = 'GCP' OR C.cloud_platform IS NULL)";
+    String sql = WORKSPACE_SELECT_SQL + " WHERE W.workspace_id = :id";
```

Comment:
I don't understand. I think that is what I have.

Reply:
oh, I meant using `workspace_id` instead of `W.workspace_id`, since there's no longer a `C.workspace_id`. Actually, you really don't need the `W` at all now, though it's just a style change